src/greatwall/gui.py

```diff
@@ -312,7 +311,6 @@ def __init__(self):
         self.transitions_list: list[QSignalTransition] = []
 
         self.greatwall = GreatWall()
-
```

Comment:
Actually this line is to separate the context when we read the code, so, please keep this line.